app/javascript/react/components/RideShowContainer.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react"
+import RideShow from './RideShow'
+
+const RideShowContainer = (props) => {
+  const [ride, setRide] = useState({})
+  const [reviews, setReviews] = useState([])
+  
+  const rideId = props.match.params.id
+  
+  const fetchRide = async () => {
+    try {
+      
+      const response = await fetch(`/api/v1/rides/${rideId}`)
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw (error)
+      }
+      const responseBody = await response.json()
+      
+      setRide(responseBody.ride)
+      setReviews(responseBody.ride['reviews'])
```

Comment:
we should be able to use dot notation here `responseBody.ride.reviews`

Reply:
has this been updated?